.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "common-dev-assets"]
+	path = common-dev-assets
+	url = https://github.com/terraform-ibm-modules/common-dev-assets.git
```

Comment:
add `branch = main` - see https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/.gitmodules .

Reply:
by default it picks primary branch, so this is `main` by default

Reply:
@ocofaigh - I've seen issues locally with git submodule update until the branch was explicit, so I don't think it hurts adding it.